package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "CheckSoldiersHealth",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.test.js",
+  "scripts": {
+    "test": "jest --collectCoverage"
+},
+"author": "",
+"license": "ISC",
+"dependencies": {
+    "@types/geojson": "^7946.0.10",
+    "axios": "^1.3.3",
+    "body-parser": "^1.20.1",
+    "cors": "^2.8.5",
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
+    "geojson": "^0.5.0",
+    "helmet": "^6.0.1",
+    "mongo": "^0.1.0",
+    "mongoose": "^6.9.1",
+    "mongoose-geojson-schema": "^2.2.4",
+    "node-ts": "^5.1.2",
+    "nodemon": "^2.0.20",
+    "ol": "^7.2.2",
+    "supertest": "^6.3.3"
+},
+"devDependencies": {
+    "@babel/core": "^7.21.0",
```

Comment:
Your aren't using babel

Reply:
There are better ways to test jest...

Try [ts-jest](https://kulshekhar.github.io/ts-jest/docs/)

[here's why](https://kulshekhar.github.io/ts-jest/docs/babel7-or-ts)

Reply:
Ok il add it in the next task